src/ray/raylet/node_manager.cc

```diff
@@ -2906,16 +2906,23 @@ MemoryUsageRefreshCallback NodeManager::CreateMemoryUsageRefreshCallback() {
           }
         }
         if (latest_worker != nullptr) {
-          RAY_LOG(INFO) << "Killing worker with the newest started task "
+          RAY_LOG(ERROR) << "Killing worker with the newest started task "
```

Comment:
This might be too spammy in some scenarios. Maybe we log ERROR only once in X times, and log INFO for the most of times?

Reply:
done